test_conformance/vulkan/vulkan_interop_common/vulkan_wrapper.cpp

```diff
@@ -19,6 +19,7 @@
 #include <Windows.h>
 #include <dxgi1_2.h>
 #include <aclapi.h>
+#include <algorithm>
```

Comment:
Same comment here - I think this should be moved out of the `_WIN32` block, and we don't need the explicit define for `NOMINMAX`.

Reply:
fixed